src/components/query-viewer/controller/query-viewer-controller.tsx

```diff
@@ -443,7 +467,7 @@ export class QueryViewerController {
       this.queryViewerServiceResponsePivotTable.emit({
         MetaData: metadata,
         metadataXML: metadataXML,
-        Properties: queryViewerBaseProperties,
+        Properties: { ...queryViewerBaseProperties, outputType: this.realType },
```

Comment:
Is `queryViewerBaseProperties` always defined? Doing `{...null, outputType: this.realType }` will throw an exception.

Reply:
I added a condition to prevent this issue